packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts

```diff
@@ -180,6 +180,17 @@ export class IncomingRequestsController extends ConsumptionBaseController {
 
         const relationship = await this.relationshipResolver.getRelationshipToIdentity(request.peer);
         // It is safe to decide an incoming Request when no Relationship is found as this is the case when the Request origins from onNewRelationship of the RelationshipTemplateContent
+
+        if (relationship?.peerDeletionInfo?.deletionStatus === PeerDeletionStatus.Deleted) {
+            return ValidationResult.error(ConsumptionCoreErrors.requests.peerIsDeleted(`You cannot decide a Request from '${request.peer.toString()}' since the peer is deleted.`));
+        }
+
+        if (relationship?.peerDeletionInfo?.deletionStatus === PeerDeletionStatus.ToBeDeleted) {
+            return ValidationResult.error(
+                ConsumptionCoreErrors.requests.peerIsToBeDeleted(`You cannot decide a Request from '${request.peer.toString()}' since the peer has the status 'ToBeDeleted'.`)
```

Comment:
I would rather describe the status then name it with `'`s. The current wording is wrong. A peer (identity / Relationship) has no status named `ToBeDeleted` which can be very confusing.

Reply:
same everywhere else